.github/workflows/ci.yml

```diff
@@ -170,6 +170,33 @@ jobs:
               '\.py$|.github/workflows/|^Dockerfile|^scripts'
           echo "::set-output name=count::$?"
         id: pyfiles
+
+  test-openapi-client-generation:
+    name: "Test OpenAPI client generation"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+      - name: "Check spec file change"
+        run: |
+          URL="https://raw.githubusercontent.com/apache/airflow/master/airflow/api_connexion/openapi/v1.yaml"
```

Comment:
We should move it to separate bash scripts. It is then much easier to test, run spellcheck and also it makes it easier to run any tests without submitting job to CI